impeller/entity/geometry.h

```diff
@@ -290,4 +294,38 @@ class RRectGeometry : public Geometry {
   FML_DISALLOW_COPY_AND_ASSIGN(RRectGeometry);
 };
 
+class PointFieldGeometry : public Geometry {
```

Comment:
TODO: figure out where to put the fallback to CPU computation. Could just add a fallback here?

Reply:
Done